### ISSUE TITLE
+config.ts getConfigData.js

### ISSUE THREAD

**Issue by enri90** [brillout/vite-plugin-ssr#816]
### Description

hello I'm trying the new version this message appears to me after configuring +config.ts

```typescript

export default {
  prerender: false,
  clientRouting: true,
  hydrationCanBeAborted: true,
  passToClient: [
    'pageProps',
    'documentProps',
    'urqlState',
    'redirectTo',
    'routeParams',
    'token',
    'session',
    'settings'
  ],
  meta: {
    documentProps: {
      env: 'server-and-client'
    },
    title: {
      env: 'server-only'
    },
    description: {
      env: 'server-only'
    },
    Layout: {
      env: 'server-and-client'
    }
  }
} satisfies Config
``` 

thx

### Error Message + Error Stack

```shell
/Users/enri/Code/urql/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:215
            (0, utils_1.assert)(configValueFiles.length === 1);
                               ^
Error: [vite-plugin-ssr@0.4.116][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Go to https://github.com/brillout/vite-plugin-ssr/issues/new and copy-paste this error. (The error's stack trace is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust.
    at resolveConfigElement (/Users/enri/Code/urql/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:215:32)
    at /Users/enri/Code/urql/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:135:35
    at Array.forEach (<anonymous>)
    at /Users/enri/Code/urql/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:134:63
    at Array.forEach (<anonymous>)
    at loadConfigData (/Users/enri/Code/urql/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:115:13)
    at async resolveConfig (/Users/enri/Code/urql/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/config/index.js:28:48)
```

**Comment by brillout** [comment]
Seems to be a duplicate of https://github.com/brillout/vite-plugin-ssr/issues/812.

**Comment by brillout** [comment]
Fix released in `0.4.117`.
